src/retryOnEmpty.ts

```diff
@@ -103,41 +98,39 @@ export function createRetryOnEmptyMiddleware({
     // create child request with specific block-ref
     const childRequest = klona(req);
     // attempt child request until non-empty response is received
-    const childResponse: PendingJsonRpcResponse<Block> = await retry(
-      10,
-      async () => {
+    try {
```

Comment:
This `try` seems to wrap the `retry`. But if we wanted to replicate the existing behavior, would we want to wrap the `request` call instead?